### ISSUE TITLE
Destroy instance -  difference between list of VMs and inside VM details

### ISSUE THREAD

**Issue by marcinpakula** [apache/cloudstack#5885]
##### ISSUE TYPE

 * Bug Report
 * Improvement Request
 * Enhancement Request
 * Feature Idea
 * Documentation Report
 * Other

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.15.2.0.
~~~

##### STEPS TO REPRODUCE

~~~ 
1. Try to destroy instance from  view list of VMs
2. Try to destroy instance from view of VM details
~~~

##### EXPECTED RESULTS

~~~
1. Possibility of expunge attached datadisk from view of list of VMs.
~~~

##### ACTUAL RESULTS

~~~
1. There is no possibility of expunge attached datadisks from options available from view of list of VMs
~~~
Destroy instance's options on form - list of VMs:
![1](https://user-images.githubusercontent.com/50416446/150531280-b420ee53-8fcb-422c-bfec-c3e14070ce5e.png)
Destroy instance's options on  VM details form:
![2](https://user-images.githubusercontent.com/50416446/150531389-ec32468d-81f4-4850-a742-e2b42dd07560.png)

**Comment by utchoang** [comment]
@marcinpakula Did you test this issue in the 4.16 version?

**Comment by sureshanaparti** [comment]
Hi @utchoang I checked with the latest 4.16, it seems the 'Data Volumes to be deleted' option is not shown in the Destroy Instance form invoked from the VMs list page. Can you check this. Thanks.

**Comment by utchoang** [comment]
@sureshanaparti cc @shwstppr @Pearl1594 In version 4.16, this form serves to multiple destroy the VM. If you need a volume option, you will need to add an option corresponding to each selected VM?
![image](https://user-images.githubusercontent.com/13766648/150744473-f37a48aa-546a-4ff8-9b03-ed46d2808805.png)

**Comment by shwstppr** [comment]
@utchoang ideally, it would be nice to have a selection for each VM. Maybe a new column to select list the selected volumes and button to open a new popup for selection in each row.
A simplistic approach can be showing volume selection like before only when a single VM is selected for destroy.
This is just my opinion :)

**Comment by sureshanaparti** [comment]
Fixed in #5893
